libraries/liveIntentIdSystem/liveIntentIdSystemShared.js

```diff
@@ -0,0 +1,201 @@
+import {UID1_EIDS} from '../uid1Eids/uid1Eids.js';
```

Comment:
I think we can drop the liveIntentId prefix from all files in libraries. They are in a folder called liveintentId anyway. So I would do:

* externalIdSystem.js
* idSystem.js
* shared.js

Reply:
Okie